src/plasoscaffolder/bll/services/file_handler.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
     """ Constructing the FileHandler"""
 
   @staticmethod
```

Comment:
If this function belongs to the class I opt to use classmethod instead. 
```
@classmethod
def create_file_path(cls, path, name, suffix):
```

If not consider to make it a regular function outside of the class.